### ISSUE TITLE
new function balance.table() or balance.plot()

### ISSUE THREAD

**Issue by graemeblair** [egap/registration#27]
That takes a realization of the data and treatment assignment (could be real treatment assignment per Macartan's example) and plots or tables balance.

**Comment by graemeblair** [comment]
Added both. commit 0598eba0fdf74a91ffad758bd014fb0a5bb34a96

See balance() function; can do plot(balance) and table(balance).

**Comment by macartan** [comment]
Can this give imbalance in terms of sd of x?

**Comment by graemeblair** [comment]
Yes, will do. Reopening this issue to make a number of changes to these functions.

**Comment by graemeblair** [comment]
The programming of the balance table and plot is a bit of a bear, so can we agree on the format before I code it up again? Here is an [example balance table](https://github.com/egap/registration/files/344/Balance.Example.pdf) that Alex and I agreed on. It drops the differences altogether, to distract the reader's attention from calculating statistical significance of the differences for each variable. Thoughts?

**Comment by graemeblair** [comment]
One more thing: note that how each variable is displayed depends on its variable type (continuous i.e. income, categorical i.e. party ID and gender, binary i.e. house ownership).

**Comment by macartan** [comment]
My sense is that we do want differences in terms of sds. Larger differences
can mean greater conditional bias which is what is important here.

Agree on focussing away from ps. But they are still here in a setting in
which there is no error in implementation of randomization.  so I woukd
drop. No clear hypothesis that they are testing.

If we do include we need to be sure they correctly take account of cluster
structure etc. Are they done with ri?

**Comment by acoppock** [comment]
One question that Graeme and I ran into is: what to do for multi-arm trials?  With three-arm, there are 3 pairs of differences to show, and with a four-arm trial it's just unmanageable.  We were thinking that the default balance table should just show means and ses?  Or we could optionally turn on a standardized difference column for two-arm trials only?

**Comment by macartan** [comment]
Can we think of it in terms of qois ?
Eg maybe 4 arms but qoi's are just t1-t2 and t3-t4 and then the relevant
balance is for those comparisons. Need to think through what woukd be
relevant balance for more complex quantities -- interactions and more...
maybe we should be thinking of something more general like what is the
expected conditional bias associated with this type of draw (devil in
details for defining type)? Or given this draw how different would a
conditional analaysis be from an unconditional one? These require real
kniwledge on relationship between xs and potential outcomesnqhich could be
asking too much.
